### PR TITLE
inclusive language & region-agnosic trial URL

### PR DESCRIPTION
**Pull request by Jessica-Katz**

### CODE REVIEW

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In addition to global accounts, directories, and subaccounts, the Cloud Foundry
...
+The ABAP environment is based on the latest ABAP platform cloud release that is also used for SAP S/4HANA Cloud. It leverages the innovations provided by SAP HANA. The software stack contains standard technology components that are familiar from the standalone Application Server ABAP. The ABAP environment supports the ABAP RESTful Application Programming Model including SAP Fiori and Core Data Services (CDS). SAP Services and APIs are offered according to a new allowlist approach. The ABAP environment provides you technical access to SAP Business Technology Platform services, such as destination service, integration, machine learning, and IoT.
```
Comment (MichaelCzcz) [inline tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:66]: I think that it should be "allow list". Please check.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md (hunk 1 above)
Comment (Jessica-Katz) [reply tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:66]: No, actually it should be "allowlisting" but the spellchecker would not let me publish that.
https://help.sap.com/doc/b0322267728e48a28b0c8ee7dd1ab4c7/1.0/en-US/Inclusive%20Language%20Guidelines.pdf

Comment (MichaelCzcz) [reply tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md:66]: @thecodester Do you have an idea on how to solve this?

**Comment by thecodester** [comment]
"allowlisting" now allowlisted (if that's a word).

P.S.: "whitelisting" is forbidden, so even if there is a technical issue with the correct word, we cannot simply default back to the disallowed word.

**Comment by Jessica-Katz** [comment]
@MichaelCzcz can we merge this?

**Pull request merged**